src/open_inwoner/templates/pages/cases/statuses.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n form_tags i18n grid_tags solo_tags link_tags button_tags icon_tags notification_tags %}
+
+{# WILL BE UPDATED WHEN status_inner IS FINISHED #}
+
+{% if case.statuses %}
+<h2 class="h2" id="statuses">{% trans 'Status' %}</h2>
+
+<aside class="status-list" aria-label="{% trans "Status lijst" %}">
+    <ul class="status-list__list">
+        {% for status in case.statuses %}
+
+            {% if forloop.last %}
+                {# Current active status class #}
+                <li class="status-list__list-item {% if case.end_statustype_data %}status--current status-list__list-item--{{ status.status_indicator }} status--open{% endif %} {# Final status class #}{% if not case.end_statustype_data %}status--final status-list__list-item--info status--open{% endif %}">
+                {% firstof case.id|slugify as id %}
+                {% firstof 'content-'|add:id|add:'-id' as content_id %}
+                    {% icon icon="circle" extra_classes="status-step" %}
+                    <div class="status-list__notification status-list__notification--{{ status.status_indicator }}">
+                        <button class="button--borderless {% if case.end_statustype_data %}status-list__button{% endif %} {# Final status class #}{% if not case.end_statustype_data %}status-list__open-button{% endif %}" aria-controls="{{ content_id }}" aria-expanded="true" id="{{ id }}">
+                            <span class="link link--bold">{{ status.label }}</span>
+                            {% if case.end_statustype_data %}{% icon icon="expand_more" icon_position="after" outlined=True %}{% endif %} </button>
+                        <div class="status-list__notification-content {% if case.end_statustype_data %}status-content--open{% endif %} {% if not case.end_statustype_data %}status-content--open{% endif %}" id="{{ content_id }}">
+                            <p class="p p-text p--compact">
+                                {# Should be ZaakTypeResultaatTypeConfig configurable description #}
+                                {# {{ status.statustype.omschrijving }} #}
+                                {{ status.status_indicator_text }}</p>
```

Comment:
Missing default: `{{ status.status_indicator_text|default:"" }}` (without this it will print `None` if there is no status indicator text)

Reply:
This status_indicator_text text will be replaced later by data from ZaakTypeResultaatTypeConfig so wille be removed anyway.